SPECS/uefistored.spec

```diff
@@ -52,8 +53,14 @@ cp %{SOURCE1} %{buildroot}%{_datadir}/varstored/
 %{_sbindir}/varstored
 %dir %{_datadir}/varstored
 %{_datadir}/varstored/PK.auth
+%{_sbindir}/secureboot-certs
 
 %changelog
+* Wed Mar 3 2021 Bobby Eshleman <bobby.eshleman@gmail.com> - 0.3.2-1
```

Comment:
The day must be on two digits: `03`

Reply:
Ah, I did not notice that was a strong convention (rpmbuild nor rpmlint complain).  Will fix.